packages/admin-ui/ui/src/i18n/index.ts

```diff
@@ -12,6 +12,10 @@ export const supportedLanguages = [
     locale: "fr",
     name: "Français",
   },
+  {
+    locale: "pt-BR",
+    name: "Português Brasileiro",
```

Comment:
Nit: Usually I think you differentiate different locales of a language as `Portuguese (Brazil)` and `Portuguese (Portugal)`, etc.

Reply:
We decided to display the name as used in the given language. We might add another prop english_name, that we can use in a tooltip or similar.

Reply:
Probably another property is a better choice. For the name I would change it to `Português (Brasil)`

Reply:
Got it thanks, I'll change the name for now, and we can think about adding the English version in a follow up 👍